### ISSUE TITLE
Update component view to display all sources for ServiceRequest, Immunization, and CareTeam

### ISSUE THREAD

**Issue by kbertodatti** [chronic-care/mcc-project#312]
Do this for 3 components (all isolated so can work in parallel):
* ServiceRequest
* Immunization
* CareTeam

**Comment by kbertodatti** [comment]
@savanahmueller find comments from dan in slack and paste it in.

**Comment by savanahmueller** [comment]
@drbgfc comments in slack: 
-Update any summaries with TODO comments to support the edge cases of data extracted directly from fhirDataCollection array (vs prior, fhirData object). Most of the data we work with is displayed via the 2d summary arrays which are extracted in a complex process from fhirDataCollection. The remaining data is direct and only applies to a few summary list components. Each case of extraction within each summary list component is likely unique, and displaying limited data. All that needs to be updated is the display code within the summary list component file. Everything to support it is already in place. So, it's just a display of it and should not be too complex/a small amount of code in a few view files.
--Note: Summary list component files that DON'T need fhirDataCollection view updates as all data displayed is derived from summary matrix:
---MedicationList
---ConditionList
---GoalList
---LabResultList
---VitalsList
--Updates ARE needed in:
---CareTeamList
---ImmunizationList
---ServiceRequestList

**Comment by kbertodatti** [comment]
updates from @sravandigitalhie : ServiceRequest and CareTeam are done. Will post to develop branch after he completes Immunization.

After this, @drdavec will create new deployment to Github Pages so we can all test.

**Comment by drdavec** [comment]
Reviewed, tested and merged into develop branch.

To test, I logged into three sandboxes: MELD, Epic, and Cerner.

Immunizations shows data from both MELD and Epic
ServiceRequest shows data from both MELD and Cerner
CareTeam shows practitioners from all 3
